### DISCUSSION TITLE
How to get orphans's vertex through nGQL

### DISCUSSION THREAD

**Discussion by Milittle** [vesoft-inc/nebula#4316] [Q&A] [answered]
How to get orphans' vertex through nGQL.
version: 3.0.1, thx

**Comment by wey-gu** [comment, marked as answer]
```
match (v:player) where not (v:player)--() return v 
```
It may require 3.1.x

**Reply by Milittle** [reply]
Any other trick way in 3.01 can imple this request, if not, I will consider about upgrade version. thanks

**Reply by wey-gu** [reply]
I recommend an upgrade, all clients are compatible with 3.1.0(compared to 3.0.x), and the data is also compatible (thus no data migration is needed, just upgrade the binaries and restart!)

**Reply by Milittle** [reply]
Thx, indeed.
